homeassistant/components/homewizard/sensor.py

```diff
@@ -387,23 +392,6 @@ class HomeWizardSensorEntityDescription(
         has_fn=lambda data: data.monthly_power_peak_w is not None,
         value_fn=lambda data: data.monthly_power_peak_w,
     ),
-    HomeWizardSensorEntityDescription(
-        key="total_gas_m3",
-        translation_key="total_gas_m3",
-        native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
-        device_class=SensorDeviceClass.GAS,
-        state_class=SensorStateClass.TOTAL_INCREASING,
-        has_fn=lambda data: data.total_gas_m3 is not None,
-        value_fn=lambda data: data.total_gas_m3 or None,
-    ),
-    HomeWizardSensorEntityDescription(
-        key="gas_unique_id",
-        translation_key="gas_unique_id",
-        icon="mdi:alphabetical-variant",
-        entity_category=EntityCategory.DIAGNOSTIC,
-        has_fn=lambda data: data.gas_unique_id is not None,
-        value_fn=lambda data: data.gas_unique_id,
-    ),
```

Comment:
Why are these removed? Does it mean this is a breaking change?

Reply:
- total_gas_m3 is now exposed as sensor in the added device. So no breaking change expected.
- gas_unique_id is added in the name. This value never changes (a new device is added when the gas meter is replaced). We can mark this as a breaking change but it is a small one IMO.

Reply:
> gas_unique_id is added in the name

That I think is not a good idea. It is not really user friendly to squash IDs into names. Besides, once renamed, you loose track of it.

I would encourage to revert that change.

../Frenck

Reply:
Change has been reverted, by adding the unique ID as sensor in the external device.